declarations/Fairbnb.coop.json

```
@@ -5,6 +5,9 @@
       "fetch": "https://fairbnb.coop/terms-of-use/",
       "select": [
         "#main-content"
+      ],
+      "filter": [
+        "filterProtectedEmail"
```

Naming a filter function `filter` seems a bit redundant, what about something like:
```suggestion
        "unifyEmailProtectedAddresses"
```